subprojects/core/src/main/java/org/gradle/execution/plan/DefaultExecutionPlan.java

```diff
@@ -77,8 +77,8 @@
     private final TaskNodeFactory taskNodeFactory;
     private final TaskDependencyResolver dependencyResolver;
     private final NodeValidator nodeValidator;
-    private final RelatedLocations producedLocations;
-    private final RelatedLocations consumedLocations;
+    private final ExecutionNodeAccessHierarchy outputsHierarchy;
+    private final ExecutionNodeAccessHierarchy inputsHierarchy;
```

Comment:
I think it would be better to use singular in the names here.

Reply:
I changed it to singular.